pkg/validations/commandvalidations/commandvalidations.go

```diff
@@ -0,0 +1,82 @@
+package commandvalidations
+
+import (
+	"context"
+	"fmt"
+	"runtime"
+
+	"github.com/aws/eks-anywhere/pkg/cluster"
+	"github.com/aws/eks-anywhere/pkg/dependencies"
+	"github.com/aws/eks-anywhere/pkg/validations"
+	"github.com/aws/eks-anywhere/pkg/workflows/interfaces"
+)
+
+type ValidateCreateCluster struct {
+	clusterSpec    *cluster.Spec
+	deps           *dependencies.Dependencies
+	cmdValidations interfaces.Validator
+	dockerExec     validations.DockerExecutable
+}
+
+func New(clusterSpec *cluster.Spec, deps *dependencies.Dependencies, cmdValidations interfaces.Validator, dockerExec validations.DockerExecutable) *ValidateCreateCluster {
```

Comment:
instead of taking `dependencies.Dependencies` I would just take whatever this objects needs
If it becomes a lot of arguments, then feel free to create an input struct just for that

Reply:
That sounds good